corehq/apps/es/users.py

```diff
@@ -64,14 +64,8 @@ def show_only_inactive(self):
         return query.is_active(False)
 
 
-def domain(domain, allow_enterprise=False):
-    domain_list = [domain]
-    if allow_enterprise:
-        from corehq.apps.enterprise.models import EnterprisePermissions
-        config = EnterprisePermissions.get_by_domain(domain)
-        if config.is_enabled and domain in config.domains:
-            domain_list.append(config.source_domain)
-    return domains(domain_list)
+def domain(domain):
```

Comment:
It really is nice to get the enterprise logic out of here and into the `enterprise` app - thank you for cleaning this up.

Reply:
:star2: